frontend/src/app/globals.css

```
@@ -159,10 +159,10 @@
   --auth-section-padding: 4rem;
   --auth-input-height: 2.75rem;
 
-  /* Auth typography */
-  --auth-font-heading: 'Inter', -apple-system, BlinkMacSystemFont, 'SF Pro Display', system-ui, sans-serif;
-  --auth-font-body: 'Inter', -apple-system, BlinkMacSystemFont, 'SF Pro Display', system-ui, sans-serif;
-  --auth-font-brand: var(--font-nunito), sans-serif;
+  /* Auth typography — uses Geist Sans from root layout */
+  --auth-font-heading: var(--font-geist-sans), -apple-system, BlinkMacSystemFont, 'SF Pro Display', system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)